Sources/Supporting Files/macOS/SessionTools_macOS.h

```diff
@@ -0,0 +1,19 @@
+//
+//  SessionTools_macOS.h
+//  SessionTools-macOS
+//
+//  Created by Ryan Gant on 1/15/19.
+//  Copyright © 2019 CocoaPods. All rights reserved.
```

Comment:
Another copyright changed needed to use Bottle Rocket Studios

Reply:
Done.